modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
```

Comment:
This procedure is really about editing the ConfigMap so that the storageclass uses the ReadWriteMany accessMode required for live migration to function properly. Since this procedure already requires a storageclass to have had its storage settings added to the ConfigMap, I would suggest it would be better for the procedure to focus on adding the correct storage settings in the first place. Or maybe we can add an admonition or Prerequisite to the live migration assembly that links to the [existing content](https://docs.openshift.com/container-platform/4.4/cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.html) about these storage settings?

Reply:
Moved the prereq to the assembly and added the xref in the new intro.

* Before using LiveMigration, ensure that the storage class used by the 
virtual machine has a persistent volume claim (PVC) with a shared 
ReadWriteMany (RWX) access mode. 
See xref:../../cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes[Storage defaults for DataVolumes]
to ensure all your storage settings are correct.

Reply:
This has not been addressed

Reply:
From our Slack conversation of Monday June 22, it sounds like the problem here was (as below) the clarification of the task and it's reflection in the title of the module. I believe this to be clearer now that I have eliminated "enable" and changed the wording, but let's meet in some dynamic medium like Blue Jeans or Google Meet if there are still issues you feel should be addressed.